server/service/AirspaceService.js

```diff
@@ -130,6 +135,21 @@ const service = {
   },
 };
 
+/**
+ * To ensure that smaller airspaces are not overlayed by bigger ones we will sort big airspace to the beginning
```

Comment:
Hm did not really work here. The CTR should be shown, b/c its the lowest, no?
<img width="606" alt="Screenshot 2022-03-13 at 18 50 57" src="https://user-images.githubusercontent.com/35194489/158072518-ec6888a6-8554-4372-8c9a-3d91b76c8a0a.png">

Reply:
Schau dir die Implementierung an. Dann wird dir klar wieso das hier keine Auswirkung hat.

Bislang werden nur Lufträume der Klasse C und D nach vorne sortiert. Eine komplete Reihenfolge habe ich nicht implementiert, da ich es schwierig fand, dass in eine vernüftige Reihenfolge zu bringen. 

Entweder ich nehme die TMZ mit in diese "bigAirspaceClasses" mit auf oder ich schlage folgende Reihenfolge vor:
Q, R, W, CTR, RMZ, TMZ, D, C

Reply:
Wie wäre es, wenn wir dir nach Untergrenze sortieren? Für uns am Interessantesten wird immer der Unterste sein.